src/components/multiStepBar/MultiStepBar.tsx

```diff
@@ -0,0 +1,67 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import {IClassName} from '../../utils/ClassNameUtils';
+import {ITooltipProps, Tooltip} from '../tooltip/Tooltip';
+
+export type IMultiStepState = 'to-do' | 'doing' | 'done' | 'error' | 'warning';
+
+export interface IStep {
+    state: IMultiStepState;
+    text?: string;
+    tooltip?: ITooltipProps;
+}
+
+export interface IMultiStepBarProps {
+    className?: IClassName;
+    small?: boolean;
+    separator?: boolean;
+    inProgressAnimation?: 'slide' | 'shine';
```

Comment:
Should you allow empty string as a value?

Reply:
It is optional, so you can just omit it 🤔

But I guess we better put it just it case